doc/content/reference/telemetry/cli.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "CLI Telemetry"
+description: "Description of everything that the telemetry procedure collects regarding the CLI's user"
+aliases: "/telemetry/cli"
+---
+
+{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
```

Comment:
```suggestion
{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** telemetry data, which is used to help us understand how users interact with our product. For example, the current data helps to have a better insight on what type of machines the CLI is more commonly used on. Telemetry collection is enabled by default and does not collect personally identifiable information. The following section section explains what information is collected and how it is anonymised.
```

Reply:
> Telemetry collection is enabled by default and does not collect personally identifiable information.

I think this sentence is worth stressing somehow. Maybe add it as a note or in bold?

Reply:
> Telemetry collection is enabled by default

In the CLI page I believe its worth making only this bold, as the rest of the phrase is already stressed in the first sentence of the paragraph.
>{{% tts %}}\'s [CLI]({{<ref "/reference/cli">}}) collects **completely anonymous** 

That way we would have something like the image below, what you think @nejraselimovic?
![image](https://github.com/TheThingsIndustries/lorawan-stack-docs/assets/28912302/219d6ce9-3fbf-4e11-9806-d381d5ca03ed)

Reply:
Actually changed in a way where only the words `enabled` and `not` are bold. I'm marking this conversation as resolved and updating the PR description with the new screenshots